goLive.php

```diff
@@ -43,12 +43,12 @@
     $stream = $ig->live->create();
     $broadcastId = $stream->getBroadcastId();
     $ig->live->start($broadcastId);
-    // Switch from RTMPS to RTMP upload URL, since RTMPS doesn't work well.
-    $streamUploadUrl = preg_replace(
+    // Switch from RTMPS to RTMP upload URL, since RTMPS doesn't work well. "no need to switch it port 80 not working"
+    /*$streamUploadUrl = preg_replace(
         '#^rtmps://([^/]+?):443/#ui',
         'rtmp://\1:80/',
         $stream->getUploadUrl()
-    );
+    );*/
 
     //Grab the stream url as well as the stream key.
     $split = preg_split("[".$broadcastId."]", $streamUploadUrl);
```

Comment:
$streamUploadUrl does not exist as the previous code commented out. Add 
$streamUploadUrl =  $stream->getUploadUrl();